assets/base.css

```diff
@@ -3353,6 +3353,11 @@ details-disclosure > details {
     transform: rotate(0.5deg); /* Less intense rotation for collage items. */
   }
 
+  .animate--hover-3d-lift .grid__item:hover,
+  .animate--hover-3d-lift .collage__item:hover {
+    z-index: 1; /* Make sure the hovered card is the topmost card. */
+  }
```

Comment:
As noted in the PR description, I would've preferred to target `card-wrapper:hover` as usual, but changing the z-index there won't actually lift the card + shadow above it's neighbors. I had to target the level above like this to make that happen.